README.md

```diff
@@ -19,7 +19,7 @@ To run the pipeline end-to-end but only generate data for Connecticut state / co
 # Fetches latest scraped data from can-scrapers and combines all data sources
 # into a combined dataset, runs all filters, etc.  Adding --no-refresh-datasets
 # will make this much faster but skips fetching / combining latest datasets.
-./run.py data update --state CT
```

Comment:
the README was wrong as of a change a long time ago, the correct flag is `states`, but this is unrelated to this PR